src/expandableCalendar/WeekCalendar/index.tsx

```diff
@@ -150,7 +150,6 @@ class WeekCalendar extends Component<WeekCalendarProps, State> {
         current={item}
         firstDay={firstDay}
         style={this.getWeekStyle(this.containerWidth, style)}
-        markedDates={markedDates}
```

Comment:
I see u remove markDates completely, don't we still need them for general markings?

Reply:
They will be passed in `...others`